website/package.json

```diff
@@ -2,10 +2,10 @@
   "scripts": {
     "prestart": "node lib/build.js",
     "start": "docusaurus-start",
-    "build": "yarn prestart && docusaurus-build",
-    "publish-gh-pages": "yarn prestart && docusaurus-publish",
+    "build": "node lib/build.js && docusaurus-build",
+    "publish-gh-pages": "node lib/build.js && docusaurus-publish",
     "write-translations": "docusaurus-write-translations",
-    "version": "yarn prestart && docusaurus-version",
+    "version": "node lib/build.js && docusaurus-version",
```

Comment:
Should we remove the "prestart" script then?

Reply:
@edemaine It is run before `start`, but since it's single command and to be consistent with others, we can move `node lib/build.js` to `start`. What do you think?

Reply:
I'd say either yes, move it into `start`, or just revert that last commit and have everyone call `yarn prestart`.  Slight reference for the latter.

Reply:
I've committed the request change in #1584. I'll commit the changes here, if it gets disapproved.